SignalRServiceBenchmarkPlugin/common/ThreadSafeRandom.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace Common
+{
+    public static class ThreadSafeRandom
+    {
+        [ThreadStatic] private static Random Local;
```

Comment:
`ThreadLocal<Random>`?

Reply:
Since the shuffle is only in one thread, I use Random instead.